src/Services/FhirResourcesService.php

```diff
@@ -97,13 +99,18 @@ public function createPatientResource($resourceId = '', $data = '', $encode = tr
         }
     }
 
-    public function createPractitionerResource($id = '', $data = '', $encode = true)
+    public function createPractitionerResource($provider_id = '', $encode = true)
     {
+        if(!$provider_id) {
```

Comment:
```suggestion
        if (!$provider_id) {
```
For Happy Travis 😀

Reply:
yeah thanks. forgot to scan last night. too tired and too old:) thanks though